pkg/generic/binarythrift_codec_test.go

```diff
@@ -98,14 +98,71 @@ func TestBinaryThriftCodec(t *testing.T) {
 	test.Assert(t, method == "mock", method)
 }
 
+func TestBinaryThriftCodecExceptionError(t *testing.T) {
+	ctx := context.Background()
+	btc := &binaryThriftCodec{thriftCodec}
+	cliMsg := &mockMessage{
+		RPCInfoFunc: func() rpcinfo.RPCInfo {
+			return newEmptyMethodRPCInfo()
+		},
+		RPCRoleFunc: func() remote.RPCRole {
+			return remote.Server
+		},
+		MessageTypeFunc: func() remote.MessageType {
+			return remote.Exception
+		},
+	}
+
+	rwbuf := remote.NewReaderWriterBuffer(1024)
+	// test data is empty
+	err := btc.Marshal(ctx, cliMsg, rwbuf)
+	test.Assert(t, err != nil, err)
+	cliMsg.DataFunc = func() interface{} {
+		return &remote.TransError{}
+	}
+
+	// test Exception
+	// empty method
```

Comment:
for this assertion, it may don't matter with Exception? just test when method is empty